src/pdn/src/via.cpp

```diff
@@ -2851,11 +2851,15 @@ void Via::writeToDb(odb::dbSWire* wire,
     const TechLayer tech_layer(lower_->getLayer());
     int x = 0;
     int y = 0;
+    int ripup_count = 0;
     for (auto* via : ripup_vias) {
       int via_x, via_y;
-      via->getViaXY(via_x, via_y);
-      x += via_x;
-      y += via_y;
+      if (via->getBlockVia() != nullptr || via->getTechVia() != nullptr) {
```

Comment:
When would a via have neither of these conditions?

Reply:
The SBoxes also contain patch metals (in the case of a via stack), this caused an assert error when compiled with debug.

Reply:
So 'via' isn't necessarily a via?  That's a bit confusing.

Reply:
Yeah, I can change the variable name (it used to just be vias) :-)